apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/Constants.java

```diff
@@ -127,7 +129,11 @@
 
     public static final String ALPHABET = "0123456789abcdefghijklmnopqrstuvwxyz";
     public static final String ALLOWED_CURRENCY_CODE_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-    
+
+    //Eth/Pax
+    public static final BigInteger GAS_LIMIT_ETHER_TX = BigInteger.valueOf(21_000);
```

Comment:
Maybe move dex constants to separate class?

Reply:
It will make sense when we move dex and eth in to another module. Because if we will create new class for dex constants now it makes mess.